### PR TITLE
Added PHP is_finite() method to math functions in the documentation

### LINKED ISSUE #2399: [Term Entry] PHP Math Functions is_finite()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `is_finite()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/is-finite/is-finite.md` (Note the path uses dashes, not underscores.)

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by LewisMK**
### Description

Added the PHP is_infinite() method to the documentation of terms in PHP math functions.  Closes #2399 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`is_finite()`** method evaluates whether the value is finite or not and returns a *boolean* value.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/is-finite/is-finite.md:17]:
```suggestion
The **`is_finite()`** function evaluates whether the value is finite or not and returns a boolean value.
```

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Syntax
+
+```pseudo
+$result = is_finite(value);
+```
+
+Returns `1` if `value` is finite, else returns `false/nothing`
+
+## Example
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/is-finite/is-finite.md:29]: This example doesn't produce any output or do anything with the result. Can we have a slightly more complex example, with a sentence introducing it and summarizing what it does?

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<?php
+  $result = is_finite(100);
+?>
+```
+
+## Codebyte Example
+
+The codebyte below deploys the `is_finite()` method on the values of `100` and `log(0)` to evaluate whether they are `finite`:
+
+```codebyte/php
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/is-finite/is-finite.md:40]: Using echo like this doesn't produce an expected result `echo is_finite(log(0));` doesn't produce any output. The output of the codebyte as written is `1<br>`

Perhaps use some conditionals to print out if something is finite or not.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@LewisMK I did a first review and left a few comments.

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md (hunk 3 above)
Comment (LewisMK) [reply content/php/concepts/math-functions/terms/is-finite/is-finite.md:40]: Thank you for your feedback. I will address all concerns before the next submission.

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md (hunk 2 above)
Comment (LewisMK) [reply content/php/concepts/math-functions/terms/is-finite/is-finite.md:29]: Okay.

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@LewisMK looks good for a second review.

**Comment by LewisMK** [comment]
@SSwiniarski thank you for your guidance and input

**Comment by SSwiniarski** [comment]
@LewisMK if you're planning a new PR with `is_infinite()` you need to create a fresh branch off of main for it so it doesn't appear in this PR.

**Comment by LewisMK** [comment]
@SSwiniarski thanks for the correction. I am a new contributor so it's great to learn how Git works.

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+```php
+<?php
+  $result = is_finite(100);
+  echo $result; //this will output '1', indicating True
+?>
+```
+
+## Codebyte Example
+
+The codebyte below deploys the `is_finite()` function on the values of `100` and `log(0)` to evaluate whether they are `finite`:
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-finite/is-finite.md:41]:
```suggestion
The codebyte below uses the `is_finite()` function on the values of `100` and `log(0)` to evaluate whether they are finite:
```

File: content/php/concepts/math-functions/terms/is-finite/is-finite.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  $test_value2 = log(0);
+
+//Use the if-else conditional to check whether the test values are finite or not
+
+  if (is_finite($test_value1)) {
+    echo "Test value 1 is finite.";
+  } else {
+    echo "Test value 1 is not finite.";
+  }
+  echo "\n"; //separate the output for the two test values using a new line
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-finite/is-finite.md:57]:
```suggestion
  echo "\n"; // Separate the output for the two test values using a new line
```

**Review by yangc95 — CHANGES_REQUESTED** [review]
Hey @LewisMK I left a few comments and suggestions that need to be addressed. Great work on these terms! Let me know once you've taken everything into consideration and let me know if you need any assistance 🙂

**Comment by LewisMK** [comment]
@yangc95 I have committed all suggested changes. Please check and inform me if more are required.

**Comment by LewisMK** [comment]
@SSwiniarski oh noo. I deleted that repo to make another fork for a different submission. I was not aware this would happen. So sorry, what can I do?

**Comment by SSwiniarski** [comment]
The only thing you can do is resubmit it. You don't need a new fork for every submission. You should have one fork from main, and each PR should be a separate branch off of that one main fork.

When you resubmit, comment and tag me and @yangc95 to let us know we've reviewed this before.

**Pull request closed without merging**